azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeTelecomManagerOptions.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * CallCompositeTelecomManagerOptions {@link CallCompositeTelecomManagerOptions}.
+ */
+public final class CallCompositeTelecomManagerOptions {
+    private final CallCompositeTelecomManagerIntegration telecomIntegration;
+
+    /**
+     * Create {@link CallCompositeTelecomManagerOptions}.
+     *
+     * @param telecomIntegration telecom integration.
+     */
+    public CallCompositeTelecomManagerOptions(final CallCompositeTelecomManagerIntegration telecomIntegration) {
```

Comment:
Can this be null? And what would be the fallback strategy there?

Reply:
This is Enum.